### ISSUE TITLE
App won't restart on Ubuntu 20.04

### ISSUE THREAD

**Issue by bjohas** [notion-enhancer/notion-repackaged#52]
Hello,

when I run the app, and then close and try to start again, the app won't start.

I get

```
Error: Error: ENOENT: no such file or directory, open '/opt/Notion Enhanced/resources/app-update.yml'
```

Not sure if it's relevant. Any ideas?

**Comment by dieggpereira** [comment]
@bjohas I noticed a similar problem here. My solution was closing always with Ctrl + Q. I don't know what is the real problem, but here it was solved with that.

**Comment by ashleywaite** [comment]
Also on Ubuntu 20.04 and that specific error appears for me whether it ran successfully or not.

Check if it's actually closing to system tray.
When it's present in system tray clicking the regular launch shortcut again _doesn't work_ and it looks like nothing happens because it's _already running_.

This is definitely not expected behaviour and should be considered a bug IMO.
Clicking on the app icon looks like it's doing something for a moment, then nothing happens.
Likewise, using 'new window' from the system tray does not open a new window for me ever.

Using 'toggle visibility' from the system tray does show the window though, relaunch also works, as does simply killing all background notion processes with something like `killall -r "notion-all*"` and opening it again freshly.

I did get this error on terminal though when I tried to run it while it was already running and I'd closed the existing window:

```js
{
    name: 'TypeError',
    message: 'focused_window.isMaximized is not a function',
    stack: 'TypeError: focused_window.isMaximized is not a function\n' +
      '    at Object.__exports.createWindow (/opt/Notion Enhanced/resources/app/embedded_enhancer/mods/core/createWindow.js:42:43)\n' +
      '    at handleActivate (/opt/Notion Enhanced/resources/app/main/main.js:71:36)\n' +
      '    at startup (/opt/Notion Enhanced/resources/app/main/main.js:149:5)\n' +
      '    at processTicksAndRejections (internal/process/task_queues.js:97:5)'
}
```

Which would probably explain why none of those methods work.
